mcp/CMakeLists.txt

```diff
@@ -200,71 +210,26 @@ ExternalProject_Add(soem
     URL ${PROJECT_SOURCE_DIR}/../external_libs/soem
     INSTALL_COMMAND "")
 
-include_directories(${PROJECT_SOURCE_DIR}/../external_libs/soem/soem 
-    ${PROJECT_SOURCE_DIR}/../external_libs/soem/osal
-    ${PROJECT_SOURCE_DIR}/../external_libs/soem/oshw)
-
-# add katcp library
-set(KATCP_LIBRARY ${PROJECT_BINARY_DIR}/katcp/libkatcp.a)
-ExternalProject_Add(katcp 
-    PREFIX ${CMAKE_CURRENT_BINARY_DIR}/katcp 
-    BINARY_DIR ${PROJECT_BINARY_DIR}/katcp 
-    URL ${PROJECT_SOURCE_DIR}/../external_libs/katcp
-    INSTALL_COMMAND "")
-
-include_directories(${PROJECT_SOURCE_DIR}/../external_libs/katcp)
-
-# build socat
-ExternalProject_Add(socat
```

Comment:
Can remove `socat`: only needed by `roach`

Reply:
+1 to this